python/lsst/ts/wep/task/calcZernikesTask.py

```diff
@@ -113,7 +113,9 @@ class CalcZernikesTaskConfig(
         doc="How to select donut stamps.",
     )
     doDonutStampSelector = pexConfig.Field(
-        doc="Whether or not to run donut stamp selector.",
+        doc="Whether or not to run donut stamp selector."
+        + "If this is False, then we do not get donutQualityTable."
+        + "(The default is True)",
```

Comment:
I think it would be helpful to mention the `doSelection` option inside `donutStampSelector` here.